ct/rdtclient.sh

```diff
@@ -54,17 +54,33 @@ function default_settings() {
 
 function update_script() {
 header_info
+check_container_storage
+check_container_resources
 if [[ ! -d /opt/rdtc/ ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
-msg_info "Updating $APP"
+msg_info "Stopping ${APP}"
 systemctl stop rdtc
+msg_ok "Stopped ${APP}"
+
+msg_info "Updating ${APP}"
+if dpkg-query -W dotnet-sdk-8.0 >/dev/null 2>&1; then
```

Comment:
We use `apt-get` to install the package, should use the same when searching for it. 

Also not sure we should actually handle breaking changes and fixing dependencies like this in the update function. It can end up with a lot of `if` conditions. Can't we just fix `rdtclient-instal.sh``and mark it as a breaking change? Then people that wants to update from old script can install the new dependency manually

Reply:
This makes no functional difference, except that it is easier to read and understand. The install is still done with apt-get, only the query for the package is done with dpkg. 

```bash
if apt list --installed dotnet-sdk-8.0 2>/dev/null | grep -q "^dotnet-sdk-8.0/"; then
    apt-get remove --purge -y dotnet-sdk-8.0
    apt-get install -y dotnet-sdk-9.0
fi
``` 

That could be done, but I'm already seeing the first bug reports because the front end doesn't work.

In addition, probably nobody looks at the documentation. I would probably set myself a reminder and remove this line of code in 6 months. Whoever still has the problem then is long overdue either way :-D

Reply:
Ok we can add it if you want. Then make a task in the github project to remove it in some months, so it is remembered. Also have to be marked as a breaking change when it is removed later then.